LayoutTests/http/tests/site-isolation/fullscreen-escape-key.html

```diff
@@ -0,0 +1,21 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<script>
+    addEventListener("message", (event) => {                
+        if (event.data.startsWith('Entered') && window.testRunner) { eventSender.keyDown("escape"); }
+        if (event.data.startsWith('Exited') && window.testRunner) { testRunner.notifyDone(); }
+    });
+
+    testRunner.dumpFullScreenCallbacks();
+
+    function test() {
+        document.getElementById("frame").focus();
+        if (window.testRunner) {
+            testRunner.waitUntilDone();
+            testRunner.dumpAsText()
+        }
+    }
+
+</script>
+<iframe onload="test()" id='frame' allowfullscreen src="http://localhost:8000/site-isolation/resources/fullscreen-escape-key.html" frameborder=0></iframe>
```

Comment:
```suggestion
<iframe onload="test()" id="frame" allowfullscreen src="http://localhost:8000/site-isolation/resources/fullscreen-escape-key.html" frameborder=0></iframe>
```

Reply:
I wonder if you could also just use `./resources/fullscreen-escape-key.html` as src

Reply:
This doesn't expose the bug unfortunately.